enhancements/two-nodes-openshift/2no.md

```diff
@@ -73,31 +73,35 @@ This requires our solution to provide a management experience consistent with "n
 
 ### User Stories
 
-* As a large enterprise with multiple remote sites, I want a cost-effective OpenShift cluster solution so that I can manage containers without the overhead of a third node.
-* As a support engineer, I want a safe and automated method for handling the failure of a single node so that the downtime of the control-plane is minimized.
-* As an enterprise running workloads on a minimal OpenShift footprint, I want to minimize time-to-recovery and data loss for my workloads when a node fails.
+* As a solutions architect for a large enterprise with multiple remote sites, I want a cost-effective OpenShift cluster solution so that I can manage containers without the overhead of a third node.
+* As a solutions architect for a large enterprise running workloads on a minimal OpenShift footprint, I want to leverage the full capacity of both control-plane nodes to run my workloads.
+* As a solutions architect for a large enterprise running workloads on a minimal OpenShift footprint, I want to minimize time-to-recovery and data loss for my workloads when a node fails.
+* As an OpenShift cluster administrator, I want a safe and automated method for handling the failure of a single node so that the downtime of the control-plane is minimized and the cluster fully recovers.
 
 ### Goals
 
 * Provide a two-node control-plane for physical hardware that is resilient to a node-level failure for either node
 * Provide a transparent installation experience that starts with exactly 2 blank physical nodes, and ends with a fault-tolerant two-node cluster
 * Prevent both data corruption and divergent datasets in etcd
-* Minimize recovery-caused unavailability. Eg. by avoiding fencing loops, wherein each node powers cycles its peer after booting, reducing the cluster's availability.
+* Minimize recovery-caused unavailability (e.g. by avoiding fencing loops, wherein each node powers cycles its peer after booting, reducing the cluster's availability)
 * Recover the API server in less than 120s, as measured by the surviving node's detection of a failure
 * Minimize any differences to existing OpenShift topologies
-* Avoid any decisions that would prevent future implementation and support for upgrade/downgrade paths between two-node and traditional architectures
-* Provide an OpenShift cluster experience that is similar to that of a 3-node hyperconverged cluster but with 2 nodes
+* Provide an OpenShift cluster experience as similar to that of a 3-node hyperconverged cluster as can be achieved accepting the resiliency compromises of having 2 nodes
+* Minimize scenarios that require manual intervention to initiate cluster recovery
+* Add a new `ControlPlaneTopology` enum in the Infrastructure API
```

Comment:
Nit: I would think this one is considered more of an implementation detail than a goal

Reply:
Yeah, I think there's probably a way to rephrase this that captures the point of that change.

Specifically, the goal is to provide a mechanism for cluster operators to respond to the limitations of the current topology - which is more than just the core operators but also applies to OLM.

Reply:
Took a pass at cleaning this up; let me know what you think :)

Reply:
Looks great :+1: